cmd/local/local.go

```diff
@@ -419,6 +426,16 @@ func runLocal(cmd *cobra.Command, args []string) error {
 		progressPrinter.IncrementTracker("step-telemetry", 1)
 	}
 
+	_, _, err = pkg.ExecShellReturnStrings(config.KubectlClientPath, "--kubeconfig", config.KubeConfigPath, "-n", "argocd", "apply", "-f", fmt.Sprintf("%s/gitops/ingressroute.yaml", config.K1FolderPath))
+
+	if err != nil {
+
+		log.Printf("failed to create ingress route to argocd: %s", err)
+
+	}
+
+	_, _, _ = pkg.ExecShellReturnStrings(config.KubectlClientPath, "--kubeconfig", config.KubeConfigPath, "-n", "argocd", "delete", "ingress", "argocd-server")
```

Comment:
the 3rd return value is an error, it would be nice to log it, like:
```
log.Error().Err(err).Msg("")
```

it will generate a red log line with error level